onnxruntime/core/graph/graph.cc

```diff
@@ -3067,66 +3067,6 @@ ONNX_NAMESPACE::GraphProto Graph::ToGraphProto() const {
   return result;
 }
 
-ONNX_NAMESPACE::GraphProto Graph::ToGraphProtoWithExternalInitializers(const std::string& external_file_name,
-                                                                       size_t initializer_size_threshold) const {
-  GraphProto result;
-  ToGraphProtoInternal(result);
-  const auto& model_path = ModelPath();
-
-  std::ofstream external_stream(external_file_name, std::ofstream::out | std::ofstream::binary);
-  ORT_ENFORCE(external_stream.is_open());
-  int64_t external_offset = 0;
-
-  // Add the initializers to the result graph.
-  const auto sparse_end = sparse_tensor_names_.end();
-  for (const auto& initializer : graph_proto_->initializer()) {
-    if (sparse_end != sparse_tensor_names_.find(initializer.name())) {
-      // Sparse tensors are added to the ONNX file.
-      auto& sparse_initializer = *result.add_sparse_initializer();
-      auto status = utils::DenseTensorToSparseTensorProto(initializer, model_path, sparse_initializer);
-      ORT_ENFORCE(status.IsOK(), "Failed to convert dense initializer to sparse");
-    } else {
-      // Dense tensors larger than the threshold are added to the external file.
-      TensorProto* output_proto = result.add_initializer();
-
-      size_t tensor_bytes_size = 0;
-      std::unique_ptr<uint8_t[]> raw_data;
-      ORT_THROW_IF_ERROR(utils::UnpackInitializerData(initializer, Path(), raw_data, tensor_bytes_size));
```

Comment:
1. Here it passes an empty string to the UnpackInitializerData function.  Because it misses key information, UnpackInitializerData can't work properly.

2. I don't understand why it calls UnpackInitializerData. Here we should pack the initializer into raw data format, not unpack.

Reply:
onnxruntime_graph files shouldn't call functions in onnxruntime_framework.lib.  Usually it will result a link error. I'm curious why it didn't happen.